x-pack/plugins/maps/public/classes/layers/wizards/file_upload_wizard/config.tsx

```diff
@@ -12,6 +12,7 @@ import { ClientFileCreateSourceEditor, UPLOAD_STEPS } from './wizard';
 import { getFileUpload } from '../../../../kibana_services';
 
 export const uploadLayerWizardConfig: LayerWizard = {
+  id: 'uploadGeojson',
```

Comment:
rename `uploadGeojson` to `uploadGeoFile` since this wizard is for geojson and shapefile. Also, make a constant and use in registration